UKHomeOffice/sgmr-service
# NMSW-1380 Move regulatory and landing pages to page folders & rename where needed

### AC / Description of changes

Tidying up ahead of new feature builds
- move landing files out of components into landing folder
- rename Help to VoyageStart for the one that starts a voyage and move to voyage folder
- move other regulatory pages to regulatory folder


### To Test

- while signed out click `Help` in footer
- > Help page should load
- sign in
- click `Help` in footer
- > same page should load but now `voyage plans` and `help` words should be links
- click `Help` in nav
- > a different help page should load with title `What to do if you cannot access Tell Border Force and HMRC you are sailing to or from the UK in a pleasure craft`
- click on `Voyage plans` in nav
- click on `Create new voyage`
- > a start page should load titled `Telling us about your voyage plan`

### Notes

---

**Remember to merge to feature branches not `master`.**
